res/values/themes.xml

```diff
@@ -137,4 +137,10 @@
+<style name="Lightv2" parent="@style/BaseTheme"><item name="android:isLightTheme">true</item><item name="colorKeyboard">#fff2f5ff</item><item name="colorKey">#ffffffff</item><item name="colorKeyActivated">#ffc2e7ff</item><item name="colorLabel">#ff222222</item><item name="colorLabelActivated">#ff000000</item><item name="colorLabelLocked">#ffffffff</item><item name="secondaryLightOffset">0.0</item><item name="colorSubLabel">#ff333333</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorLeft">#ffcccccc</item><item name="keyBorderColorTop">#ffeeeeee</item><item name="keyBorderColorRight">#ffcccccc</item><item name="keyBorderColorBottom">#ffaaaaaa</item><item name="emoji_button_bg">?colorKeyActivated</item><item name="emoji_color">#ff000000</item></style>
+<style name="Material" parent="@style/BaseTheme"><item name="android:isLightTheme">true</item><item name="colorKeyboard">#ffeeeeef</item><item name="colorKey">@android:color/white</item><item name="colorKeyActivated">#ff2962ff</item><item name="colorLabel">@android:color/black</item><item name="colorLabelActivated">@android:color/white</item><item name="colorLabelLocked">@android:color/black</item><item name="secondaryLightOffset">0.0</item><item name="colorSubLabel">#ff555566</item><item name="keyBorderRadius">6.0dip</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorBottom">#ff651fff</item><item name="emoji_button_bg">?colorKeyboard</item><item name="emoji_color">?colorLabel</item></style>
+<style name="Desert" parent="@style/BaseTheme"><item name="android:isLightTheme">true</item><item name="colorKeyboard">#ff4db6ac</item><item name="colorKey">#ffe0f2f1</item><item name="colorKeyActivated">#ff00695c</item><item name="colorLabel">#ff000000</item><item name="colorLabelActivated">#ffffffff</item><item name="colorLabelLocked">#ff64ffda</item><item name="secondaryLightOffset">0.0</item><item name="colorSubLabel">#ff004d40</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorLeft">#00f0f0f0</item><item name="keyBorderColorTop">#00f0f0f0</item><item name="keyBorderColorRight">#00eeeeee</item><item name="keyBorderColorBottom">#00eeeeee</item><item name="emoji_button_bg">?colorKeyActivated</item><item name="emoji_color">#ff000000</item></style>
```

Comment:
Desert and Jungle looks nice :) But they seem swapped:

Jungle:
![shot](https://github.com/Julow/Unexpected-Keyboard/assets/2310568/5f72a62b-4512-4eb7-b689-3b7d4d2a685a)

Desert:
![shot](https://github.com/Julow/Unexpected-Keyboard/assets/2310568/1b0af0cf-2db6-4fde-ba6e-a6dca9ca8d66)

Reply:
EXPLANATION plis

Reply:
Jungle looks yellow and desert looks green. This seems backward.

Reply:
> Jungle looks yellow and desert looks green. This seems backward.

._.XD

Reply:
> Jungle looks yellow and desert looks green. This seems backward.

I'm getting blinder every day